  INFO root [2018-06-17 12:15:44,365] - 
  INFO root [2018-06-17 12:15:44,433] - -----------------------------------------------------------------------------------------


  INFO root [2018-06-17 12:15:45,402] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-06-17 12:15:45,403] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-06-17 12:15:45,403] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-06-17 12:15:45,605] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20013 )
  INFO root [2018-06-17 12:15:45,665] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20013, SOMAXCONN=128.
  INFO root [2018-06-17 12:15:45,668] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:38539 )
  INFO root [2018-06-17 12:15:45,669] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:38539, SOMAXCONN=128.
  INFO root [2018-06-17 12:15:45,669] - server-configs:
  INFO root [2018-06-17 12:15:45,669] - 	internalAddr : 192.168.31.56:38539
  INFO root [2018-06-17 12:15:45,669] - 	externalAddr : 192.168.31.56:20013
  INFO root [2018-06-17 12:15:45,669] - 	componentID : 9000
  INFO root [2018-06-17 12:15:46,544] - Components::findLogger: found logger, addr:192.168.31.56:38529
  INFO root [2018-06-17 12:15:46,545] - Components::addComponent: reset orderLog, uid=7919!
  INFO root [2018-06-17 12:15:46,545] - Components::addComponent[logger], uid=7919, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-06-17 12:15:46,783] - ---- loginapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: A16A7477E55D6AEDCD48345D2B50A665. Config: Debug X86. Built: 23:20:06 Jun 15 2018. AppID: 9000. UID: 7919. PID: 22744 ----
  INFO root [2018-06-17 12:15:46,784] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-06-17 12:15:46,833] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-06-17 12:15:46,834] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-06-17 12:15:48,775] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-06-17 12:15:48,825] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-06-17 12:15:48,898] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-06-17 12:15:49,519] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-06-17 12:15:49,520] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-06-17 12:15:49,541] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-06-17 12:15:49,660] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-06-17 12:15:49,739] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-06-17 12:15:50,396] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:20:02) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-06-17 12:15:50,397] - Script::install(): KBExtra is successfully!
S_INFO root [2018-06-17 12:15:50,458] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-06-17 12:15:50,459] - TelnetServer server is running on port 31000
  INFO root [2018-06-17 12:15:51,139] - ---- loginapp is running ----
 DEBUG root [2018-06-17 12:15:51,140] - Components::process(): Request for the process of identity...
  WARN root [2018-06-17 12:15:57,252] - Forwarding logs to logger[192.168.31.56:38529/0/0/0]...
  INFO root [2018-06-17 12:21:29,121] - 
  INFO root [2018-06-17 12:21:29,126] - -----------------------------------------------------------------------------------------


  INFO root [2018-06-17 12:21:29,128] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-06-17 12:21:29,129] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-06-17 12:21:29,129] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-06-17 12:21:29,202] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20013 )
  INFO root [2018-06-17 12:21:29,203] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20013, SOMAXCONN=128.
  INFO root [2018-06-17 12:21:29,207] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:38590 )
  INFO root [2018-06-17 12:21:29,208] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:38590, SOMAXCONN=128.
  INFO root [2018-06-17 12:21:29,208] - server-configs:
  INFO root [2018-06-17 12:21:29,208] - 	internalAddr : 192.168.31.56:38590
  INFO root [2018-06-17 12:21:29,209] - 	externalAddr : 192.168.31.56:20013
  INFO root [2018-06-17 12:21:29,209] - 	componentID : 9000
  INFO root [2018-06-17 12:21:31,370] - ---- loginapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: A16A7477E55D6AEDCD48345D2B50A665. Config: Debug X86. Built: 23:20:06 Jun 15 2018. AppID: 9000. UID: 9218. PID: 21056 ----
  INFO root [2018-06-17 12:21:31,371] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-06-17 12:21:31,372] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-06-17 12:21:31,373] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-06-17 12:21:31,570] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-06-17 12:21:31,572] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-06-17 12:21:31,616] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-06-17 12:21:31,623] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-06-17 12:21:31,624] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-06-17 12:21:31,624] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-06-17 12:21:31,626] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-06-17 12:21:31,811] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-06-17 12:21:31,997] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:20:02) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-06-17 12:21:31,998] - Script::install(): KBExtra is successfully!
S_INFO root [2018-06-17 12:21:32,003] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-06-17 12:21:32,004] - TelnetServer server is running on port 31000
  INFO root [2018-06-17 12:21:32,887] - ---- loginapp is running ----
 DEBUG root [2018-06-17 12:21:32,888] - Components::process(): Request for the process of identity...
  INFO root [2018-06-17 12:21:40,023] - Components::findComponents: find logger(1)...
  INFO root [2018-06-17 12:21:40,068] - Components::findComponents: found logger, addr:192.168.31.56:38594
  INFO root [2018-06-17 12:21:40,068] - Components::addComponent: reset orderLog, uid=9218!
  INFO root [2018-06-17 12:21:40,068] - Components::addComponent[logger], uid=9218, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-06-17 12:21:40,070] - Components::findComponents: find dbmgr(2)...
  WARN root [2018-06-17 12:21:41,177] - Forwarding logs to logger[192.168.31.56:38594/0/0/0]...
 ERROR root [2018-06-17 12:22:49,558] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-06-17 12:22:49,569] - The following logs sent to logger failed:
  WARN root [2018-06-17 12:22:49,569] - ==> [2018-06-17 12:22:49 557] TCPPacketReceiver::processPendingEvents(192.168.31.56:38594): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-06-17 12:22:49,570] - ==> [2018-06-17 12:22:49 559] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-06-17 12:28:39,292] - 
  INFO root [2018-06-17 12:28:39,293] - -----------------------------------------------------------------------------------------


  INFO root [2018-06-17 12:28:39,294] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-06-17 12:28:39,294] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-06-17 12:28:39,294] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-06-17 12:28:39,304] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20013 )
  INFO root [2018-06-17 12:28:39,304] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20013, SOMAXCONN=128.
  INFO root [2018-06-17 12:28:39,307] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:38698 )
  INFO root [2018-06-17 12:28:39,307] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:38698, SOMAXCONN=128.
  INFO root [2018-06-17 12:28:39,307] - server-configs:
  INFO root [2018-06-17 12:28:39,308] - 	internalAddr : 192.168.31.56:38698
  INFO root [2018-06-17 12:28:39,308] - 	externalAddr : 192.168.31.56:20013
  INFO root [2018-06-17 12:28:39,308] - 	componentID : 9000
  INFO root [2018-06-17 12:28:41,482] - ---- loginapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: A16A7477E55D6AEDCD48345D2B50A665. Config: Debug X86. Built: 23:20:06 Jun 15 2018. AppID: 9000. UID: 10623. PID: 18120 ----
  INFO root [2018-06-17 12:28:41,484] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-06-17 12:28:41,484] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-06-17 12:28:41,485] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-06-17 12:28:41,651] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-06-17 12:28:41,652] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-06-17 12:28:41,653] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-06-17 12:28:41,660] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-06-17 12:28:41,660] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-06-17 12:28:41,661] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-06-17 12:28:41,662] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-06-17 12:28:41,752] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-06-17 12:28:41,833] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:20:02) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-06-17 12:28:41,833] - Script::install(): KBExtra is successfully!
S_INFO root [2018-06-17 12:28:41,838] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-06-17 12:28:41,839] - TelnetServer server is running on port 31000
  INFO root [2018-06-17 12:28:42,934] - ---- loginapp is running ----
 DEBUG root [2018-06-17 12:28:42,935] - Components::process(): Request for the process of identity...
  INFO root [2018-06-17 12:28:50,079] - Components::findComponents: find logger(1)...
  INFO root [2018-06-17 12:28:50,168] - Components::findComponents: found logger, addr:192.168.31.56:38701
  INFO root [2018-06-17 12:28:50,169] - Components::addComponent: reset orderLog, uid=10623!
  INFO root [2018-06-17 12:28:50,169] - Components::addComponent[logger], uid=10623, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-06-17 12:28:50,171] - Components::findComponents: find dbmgr(2)...
  WARN root [2018-06-17 12:28:51,272] - Forwarding logs to logger[192.168.31.56:38701/0/0/0]...
  INFO root [2018-06-17 12:29:01,168] - 
  INFO root [2018-06-17 12:29:01,171] - -----------------------------------------------------------------------------------------


  INFO root [2018-06-17 12:29:01,173] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-06-17 12:29:01,173] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-06-17 12:29:01,174] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-06-17 12:29:01,215] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20013 )
  INFO root [2018-06-17 12:29:01,215] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20013, SOMAXCONN=128.
  INFO root [2018-06-17 12:29:01,223] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:38721 )
  INFO root [2018-06-17 12:29:01,223] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:38721, SOMAXCONN=128.
  INFO root [2018-06-17 12:29:01,224] - server-configs:
  INFO root [2018-06-17 12:29:01,224] - 	internalAddr : 192.168.31.56:38721
  INFO root [2018-06-17 12:29:01,224] - 	externalAddr : 192.168.31.56:20013
  INFO root [2018-06-17 12:29:01,225] - 	componentID : 9000
  INFO root [2018-06-17 12:29:03,234] - ---- loginapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: A16A7477E55D6AEDCD48345D2B50A665. Config: Debug X86. Built: 23:20:06 Jun 15 2018. AppID: 9000. UID: 10694. PID: 18724 ----
  INFO root [2018-06-17 12:29:03,235] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-06-17 12:29:03,236] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-06-17 12:29:03,237] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-06-17 12:29:03,460] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-06-17 12:29:03,462] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-06-17 12:29:03,462] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-06-17 12:29:03,469] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-06-17 12:29:03,470] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-06-17 12:29:03,470] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-06-17 12:29:03,471] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-06-17 12:29:03,567] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-06-17 12:29:03,653] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:20:02) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-06-17 12:29:03,653] - Script::install(): KBExtra is successfully!
S_INFO root [2018-06-17 12:29:03,658] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-06-17 12:29:03,659] - TelnetServer server is running on port 31000
  INFO root [2018-06-17 12:29:04,832] - ---- loginapp is running ----
 DEBUG root [2018-06-17 12:29:04,832] - Components::process(): Request for the process of identity...
  INFO root [2018-06-17 12:29:11,975] - Components::findComponents: find logger(1)...
  INFO root [2018-06-17 12:29:12,041] - Components::findComponents: found logger, addr:192.168.31.56:38718
  INFO root [2018-06-17 12:29:12,041] - Components::addComponent: reset orderLog, uid=10694!
  INFO root [2018-06-17 12:29:12,041] - Components::addComponent[logger], uid=10694, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-06-17 12:29:12,044] - Components::findComponents: find dbmgr(2)...
  WARN root [2018-06-17 12:29:13,145] - Forwarding logs to logger[192.168.31.56:38718/0/0/0]...
 ERROR root [2018-06-17 12:30:12,828] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-06-17 12:30:12,843] - The following logs sent to logger failed:
  WARN root [2018-06-17 12:30:12,844] - ==> [2018-06-17 12:30:12 826] TCPPacketReceiver::processPendingEvents(192.168.31.56:38718): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-06-17 12:30:12,844] - ==> [2018-06-17 12:30:12 830] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2018-06-17 12:30:13,215] - Components::findComponents: find dbmgr(32)...
delay time is too long, please check the dbmgr logs!
  INFO root [2018-06-17 12:34:06,630] - 
  INFO root [2018-06-17 12:34:06,631] - -----------------------------------------------------------------------------------------


  INFO root [2018-06-17 12:34:06,633] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-06-17 12:34:06,633] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-06-17 12:34:06,634] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-06-17 12:34:06,647] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20013 )
  INFO root [2018-06-17 12:34:06,647] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20013, SOMAXCONN=128.
  INFO root [2018-06-17 12:34:06,668] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:38775 )
  INFO root [2018-06-17 12:34:06,669] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:38775, SOMAXCONN=128.
  INFO root [2018-06-17 12:34:06,669] - server-configs:
  INFO root [2018-06-17 12:34:06,669] - 	internalAddr : 192.168.31.56:38775
  INFO root [2018-06-17 12:34:06,669] - 	externalAddr : 192.168.31.56:20013
  INFO root [2018-06-17 12:34:06,670] - 	componentID : 9000
  WARN root [2018-06-17 12:34:07,485] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20509, errorno=10048
  INFO root [2018-06-17 12:34:07,494] - ---- loginapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: A16A7477E55D6AEDCD48345D2B50A665. Config: Debug X86. Built: 23:20:06 Jun 15 2018. AppID: 9000. UID: 11690. PID: 19732 ----
  INFO root [2018-06-17 12:34:07,495] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-06-17 12:34:07,495] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-06-17 12:34:07,495] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-06-17 12:34:07,644] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-06-17 12:34:07,645] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-06-17 12:34:07,646] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-06-17 12:34:07,653] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-06-17 12:34:07,654] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-06-17 12:34:07,654] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-06-17 12:34:07,655] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-06-17 12:34:07,711] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-06-17 12:34:07,770] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:20:02) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-06-17 12:34:07,770] - Script::install(): KBExtra is successfully!
S_INFO root [2018-06-17 12:34:07,774] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-06-17 12:34:07,776] - TelnetServer server is running on port 31000
  INFO root [2018-06-17 12:34:08,358] - ---- loginapp is running ----
 DEBUG root [2018-06-17 12:34:08,359] - Components::process(): Request for the process of identity...
  INFO root [2018-06-17 12:34:15,635] - Components::findComponents: find logger(1)...
  INFO root [2018-06-17 12:34:15,655] - Components::findComponents: found logger, addr:192.168.31.56:38772
  INFO root [2018-06-17 12:34:15,655] - Components::addComponent: reset orderLog, uid=11690!
  INFO root [2018-06-17 12:34:15,655] - Components::addComponent[logger], uid=11690, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-06-17 12:34:15,657] - Components::findComponents: find dbmgr(2)...
  INFO root [2018-06-17 12:34:15,757] - Components::findComponents: found dbmgr, addr:192.168.31.56:38776
  INFO root [2018-06-17 12:34:15,757] - Components::addComponent: reset orderLog, uid=11690!
  INFO root [2018-06-17 12:34:15,757] - Components::addComponent[dbmgr], uid=11690, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-06-17 12:34:15,758] - Components::findComponents: found dbmgr, addr:192.168.31.56:38776
  WARN root [2018-06-17 12:34:16,758] - Forwarding logs to logger[192.168.31.56:38772/0/0/0]...
 ERROR root [2018-06-17 12:34:47,083] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-06-17 12:34:47,084] - The following logs sent to logger failed:
  WARN root [2018-06-17 12:34:47,086] - ==> [2018-06-17 12:34:47 082] TCPPacketReceiver::processPendingEvents(192.168.31.56:38772): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-06-17 12:34:47,089] - ==> [2018-06-17 12:34:47 084] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2018-06-17 12:34:47,219] - TCPPacketReceiver::processPendingEvents(192.168.31.56:38776): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-06-17 12:34:47,220] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2018-06-17 12:34:47,221] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2018-06-17 12:34:47,222] - onLoginAppShutDown()
  INFO root [2018-06-17 12:34:47,222] - Shutdowner::onShutdown: shutting down
  INFO root [2018-06-17 12:34:47,223] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2018-06-17 12:34:47,223] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-06-17 12:34:47,224] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2018-06-17 12:34:47,224] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2018-06-17 12:34:47,224] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2018-06-17 12:34:47,225] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2018-06-17 12:34:47,225] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2018-06-17 12:34:47,225] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2018-06-17 12:34:47,226] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2018-06-17 12:34:47,247] - Script::uninstall(): is successfully!
 DEBUG root [2018-06-17 12:34:47,247] - ThreadPool::destroy(): starting size 2.
  WARN root [2018-06-17 12:34:47,548] - ThreadPool::destroy(): waiting for thread(2)[0x18dd4a0,0x18dd438,], try=1
